src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLOR_COUNT = Colors.values().length;
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int colorNumber = random.nextInt(COLOR_COUNT);
+        return "" + Colors.values()[colorNumber];
```

Comment:
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)
(here `toString` will be called during concatenation to `""`)